es/mappings.json

```diff
@@ -16,7 +16,7 @@
       },
       "osm_key": {
         "type": "string",
-        "index": "no"
+        "index":"not_analyzed"
```

Comment:
I can do that. I noticed it, but didn't want to change what's already
there. I will do that today.
On Feb 7, 2015 8:50 AM, "Christoph Lingg" <notifications@github.com> wrote:

> In es/mappings.json
> <https://github.com/komoot/photon/pull/141#discussion_r24289893>:
>
> > @@ -16,7 +16,7 @@
> >        },
> >        "osm_key": {
> >          "type": "string",
> > -        "index": "no"
> > +        "index":"not_analyzed"
>
> we should add not_analyzed to osm_value too. at the moment it is on
> default: analyzed which is not what we want
> http://www.elasticsearch.org/guide/en/elasticsearch/guide/current/mapping-intro.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/komoot/photon/pull/141/files#r24289893>.
>